deploy-cluster-perf-node.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+if [ "$1" = "" ]
+then
+  echo "Usage: $0 <cluster name to be deployed>"
+  exit -1
+fi
+
+set -uo pipefail
+
+CLUSTER_NAME=$1
+CHARON_IMAGE_TAG="${2:-"latest"}"
+
+gcloud storage cp gs://charon-clusters-config/tokens/tokens.env .
+
+# override the env vars
+OLDIFS=$IFS
+IFS='
+'
+export $(< ./${CLUSTER_NAME}.env)
+export $(< ./tokens.env)
+IFS=$OLDIFS
+
+rm ./tokens.env
+
+# set current namespace
+kubectl config set-context --current --namespace=${CLUSTER_NAME}
+
+echo "deploying cluster: ${CLUSTER_NAME}"
+
+# Deploy charon nodes
+IFS=','
+read -a versions <<< "$CHARON_VERSIONS"
+node_index=0
+for version in "${versions[@]}"
+do
+export NODE_NAME="node$node_index"
+export VC_INDEX="vc$node_index"
+((node_index=node_index+1))
+done
+
+eval "cat <<EOF
+$(<./templates/web3signer-sa.yaml)
+EOF
+" | kubectl apply -f -
+
+# Deploy Validator client of required type for each charon node. 
+IFS=','
+read -a vcs <<< "$VC_TYPES"
+node_index=0
+for vc in "${vcs[@]}"
+do
+export NODE_NAME="node$node_index"
+export VC_INDEX="vc$node_index"
+if [ $vc -eq 0 ]; then
+eval "cat <<EOF
+$(<./templates/teku-vc.yaml)
```

Comment:
This needs to be templates/teku-vc-web3signer.yaml

Reply:
Pushed the change